### PR TITLE
Add TEMPO config files

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name
+tempo_weather,"Weather year used by TEMPO."
```
Comment (mooneyme) [inline standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv:2]: ... what is that weather year?

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv [hunk 2]
```diff
@@ -0,0 +1,34 @@
+id,name
+tempo_2018,2018
```
Comment (mooneyme) [inline standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv:2]: why temp_2018. Why not just 2018?

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions.toml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+Fuels used by TEMPO BEVs.
+"""
+
+[[dimensions]]
+class = "Time"
+type = "time"
+name = "Time of Week by Month"
+#start = "Jan Mon 00:00:00"
+#end   = "Dec Sun 23:59:59"
+ranges = [{start = "Apr Mon 00:00:00", end = "Apr Sun 23:00:00"}]
```
Comment (elainethale) [inline standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions.toml:90]: What happens to this? Does an arbitrary date get assigned? (Sorry, I think I am overly stuck on the fact that the docs said the data needs to be "in UTC". Maybe that's just not right?)

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv [hunk 4]
```diff
@@ -0,0 +1,34 @@
+id,name
```
Comment (elainethale) [inline standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv:1]: I think we decided not to give year ids prefixes?

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv (hunk 1 above)
Comment (daniel-thom) [reply standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv:2]: I copied these files directly from the dsgrid Standard Scenarios project.

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions.toml (hunk 3 above)
Comment (daniel-thom) [reply standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions.toml:90]: If we make a datetime from this the year will be something invalid like 1900, but we won't use it. This is only for validating the contents of the load data.

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv (hunk 2 above)
Comment (daniel-thom) [reply standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/year.csv:2]: This is whatever you worked out with Brian and Arthur. Shall I change it?

File: standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv (hunk 1 above)
Comment (daniel-thom) [reply standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv:2]: @mooneyme @elainethale Am I supposed to change something about this?

Comment (elainethale) [reply standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv:2]: Hopefully it is 2012?

**Review by elainethale — APPROVED** [review]

**Pull request merged**